marlowe-runtime/doc/ReadMe.md

```diff
@@ -1,100 +1,141 @@
+- syncs from the genesis block rapidly for test networks like `preprod` and `preview` ( N.B : currently takes about 15 hours to sync from the genesis of `mainnet`)
+- handles rollbacks in the blockchain
+- 3 APIs, each on different TCP sockets.
+	- *job* API: 
+		- submit a transaction to the local Cardano node
+	- *query* API:
+		- Query the local state of the Cardano node (e.g : system start time, the protocol parameters, or the slotting)
+		- Get the unspent UTxOs
+	- *sync* API: 
+		- streams or replays the blocks and transactions information from the Cardano node
 
-The `marlowe-history` executable provides services for querying the on-chain history of Marlowe contracts. It uses `chainseekd` to follow the progress of a set of Marlowe contract instances, and it handles rollbacks in the blockchain.
+See `chainseekd`'s [help page](chainseekd.md) or its [deployment instructions](../../marlowe-chain-sync/ReadMe.md) for more information.
 
-Three APIs, accessible over three different TCP sockets, are provided:
-- The *job* API supports two operations:
-	- Start following the history of a contract instance.
-	- Stop following the history of a contract instance.
-- The *query* API supports two types of queries:
-	- Fetch the status of all contracts followed by the service.
-	- Fetch the status of contracts in a provided set of contract identifiers.
-- The *sync* API provides several capabilities:
-	- Find a contract instance on the blockchain, starting from the genesis block.
-	- Find a contract instance on the blockchain, starting from a list of intersection points on the blockchain.
-	- Retrieve the next transactions for a contract instance on the blockchain after a specified point on the blockchain.
-	- Follow the history of a contract instance.
-	- Retrieve the current status of all of the contract instances being followed.
+### Marlowe History (`marlowe-history`) 
+
+- provides services for querying the on-chain history of Marlowe contracts, 
+- uses `chainseekd` to follow the progress of a set of Marlowe contract instances, 
+- handles rollbacks in the blockchain.
+- 3 APIs, each on different TCP sockets.
+	- *job* API:
+		- Start following the history of a contract instance.
+		- Stop  following the history of a contract instance.
+	- *query* API :
+		- Fetch the status of all contracts followed by the service.
+		- Fetch the status of contracts in a provided set of contract identifiers.
+	- *sync* API :
+		- Find a contract instance on the blockchain, starting from the genesis block.
+		- Find a contract instance on the blockchain, starting from a list of intersection points on the blockchain.
+		- Retrieve the next transactions for a contract instance on the blockchain after a specified point on the blockchain.
+		- Follow the history of a contract instance.
+		- Retrieve the current status of all of the contract instances being followed.
```

Comment:
This last one isn't a feature of the sync API

Reply:
removed `- Retrieve the current status of all of the contract instances being followed.`